scenarios/openshift/vmware_node_scenarios.yml

```diff
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=../plugin.schema.json
+- id: <node-stop/node-start/node-reboot/node-terminate>
+  config:
+    name: <node_name>                   # Node on which scenario has to be injected; can set multiple names separated by comma                           
+    label_selector: <label_selector>    # When node_name is not specified, a node with matching label_selector is selected for node chaos scenario injection 
+    runs: 1                             # Number of times to inject each scenario under actions (will perform on same node each time)                                                           
+    instance_count: 1                   # Number of nodes to perform action/select that match the label selector                                             
+    timeout: 300                        # Duration to wait for completion of node scenario injection
+    verify_session: True                # Set to True if you want to verify the vSphere client session using certificates; else False
+    skip_openshift_checks: False        # Set to True if you don't want to wait for the status of the nodes to change on openshift before passing the scenario 
```

Comment:
openshift -> OpenShift

Reply:
Thanks for spotting these! Fixed them now.